MMA Removal Utility/README.md

```diff
@@ -202,19 +202,20 @@ It is important to consider that if you choose to retain inventory or process lo
 ### **5. How much time AzTS MMA Discovery and Removal Utility will take to remove MMA agents?**
 Estimated average time required for a single run (both discovery and removal phase) of utility on a scope of approx. 100K VMs is around 30 Mins.
 
-### **6. How I can get list of all the VMs available in configured scope(s)?**
-VMs inventory is collected in Log Analytics workspace. To list all VMs discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+### **6. How I can get list of all the Azure services (VM,VMSS and Azure Arc servers) available in configured scope(s)?**
+Services inventory is collected in Log Analytics workspace. To list all services discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
 
 ``` KQL
 Inventory_CL
 | where TimeGenerated > ago(7d)
-| where ResourceType =~ "VirtualMachine"
 | summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
 | extend OSType = tostring(parse_json(Metadata_s).OSType)
```

Comment:
I hope OS type is being populated for all resource types properly.

Reply:
Yes.